### PR TITLE
Finalising phase 1 prototype

### PR DESCRIPTION
**Pull request by 100862367**
Summary
- 5 items have been placed around Gridstone
- Item list is now populated beforehand
- Pictures of item in item list will darken when items are collected.
- Items will only become visible when they are inside the radar
- Score text has been changed to indicate the number of items collected
- A pointer, which point to the nearest item, is placed at the location of the player.
- Item Spawn has been removed

Do not:
- Do not SetActive(false) inventory when play testing in editor or building to devices as that will prevent unity from loading the populated list and cause array out of range error.

Testplay via:
LocationBasedGame -> Map -> Spawn On Map (Script) -> Location String

### CODE REVIEW

File: FinderKeeper/Assets/Scripts/Direction.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        GameObject[] items;
+        items = GameObject.FindGameObjectsWithTag("Low");
+        GameObject closest = null;
+        float distance = Mathf.Infinity;
+        Vector3 position = transform.position;
+        foreach (GameObject item in items)
+        {
+            Vector3 diff = item.transform.position - position;
+            float curDistance = diff.sqrMagnitude;
+            if (curDistance < distance)
```
Comment (LMDyer) [inline FinderKeeper/Assets/Scripts/Direction.cs]: Could you explain this line please? Won't it always return true the first time?

File: FinderKeeper/Assets/Scripts/PhysControllerFP.cs [hunk 2]
```diff
@@ -1,153 +0,0 @@
-﻿using UnityEngine;
```
Comment (LMDyer) [inline FinderKeeper/Assets/Scripts/PhysControllerFP.cs]: Where was this script from?

File: FinderKeeper/Assets/Scripts/Direction.cs (hunk 1 above)
Comment (100862367) [reply FinderKeeper/Assets/Scripts/Direction.cs]: I'm assuming you are referring to line 21. As distance = infinity, it checks whether the distance between the item and the pointer is => infinity (which it shouldn't be), so yeah, it should always return true.

File: FinderKeeper/Assets/Scripts/PhysControllerFP.cs (hunk 2 above)
Comment (100862367) [reply FinderKeeper/Assets/Scripts/PhysControllerFP.cs]: I dropped it there a long long time ago when we first created the prototype. I have kind of just forgotten all about it and left it there. Will remove it.

**Pull request merged**